### DISCUSSION TITLE
handbook/enterprise_security/devsecops/process_and_landing/security_operational_and_response/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#5] [General]
# handbook/enterprise_security/devsecops/process_and_landing/security_operational_and_response/

软件研发安全过程中运营与响应，包括资产管理、漏洞管理等

https://sec.cafe/handbook/enterprise_security/devsecops/process_and_landing/security_operational_and_response/

**Comment by user1430** [comment]
对DevSecOps安全运营的一些理解：

**Reply by user1430** [reply]
1. 这是一个区别于漏洞运营的长期活动，规划这部分运营工作时注意长期主义和应急能力的结合，业务体量越大，联动业务配合难度越高，理解业务安全活动构成获取业务关键人支持是非常必要的。在这个活动里可以视为你在进行一个面向内部客户的商业推进活动，寻找业务内的安全支持人，说服业务安全决策者，获取业务安全用户的认可和好评；
